test/cmdlineargs.jl

```diff
@@ -4,8 +4,9 @@ import Libdl
 
 # helper function for passing input to stdin
 # and returning the stdout result
-function writereadpipeline(input, exename)
-    p = open(exename, "w+")
+function writereadpipeline(input, exename;
+                           stdout=nothing, stderr=nothing)
```

Comment:
Does stdout make sense here as an argument? Since this is supposed to read from stdout, what happens when you actually pass something here?

Reply:
Good point, `stdout` wasn't very useful as it was returned anyway, and passing `stdout=IOBuffer()` would screw it up.  I simply removed that argument, keeping only `stderr`.